pkg/golinters/gocritic.go

```diff
@@ -156,8 +158,11 @@ func (w *goCriticWrapper) buildEnabledCheckers(linterCtx *gocriticlinter.Context
 	return enabledCheckers, nil
 }
 
-func runGocriticOnPackage(linterCtx *gocriticlinter.Context, checks []*gocriticlinter.Checker,
-	files []*ast.File) []result.Issue {
+func runGocriticOnPackage(
+	linterCtx *gocriticlinter.Context,
+	checks []*gocriticlinter.Checker,
+	files []*ast.File,
+) []result.Issue {
```

Comment:
```suggestion
func runGocriticOnPackage(linterCtx *gocriticlinter.Context, checks []*gocriticlinter.Checker, files []*ast.File) []result.Issue {
```

Reply:
131 symbol per line

UPD: oh, ok, we have the 140 limit oO